praw/models/reddit/comment.py

```diff
@@ -67,6 +69,35 @@ def id_from_url(url):
             raise ClientException("Invalid URL: {}".format(url))
         return parts[-1]
 
+    @classmethod
+    def _objectify(cls, reddit, data):
+        key = "author"
+        item = data.get(key)
+        if isinstance(item, string_types):
+            data[key] = (
+                None
+                if item in ("[deleted]", "[removed]")
+                else Redditor(reddit, name=item)
+            )
+        elif isinstance(item, Redditor):
+            item._reddit = reddit
+
+        key = "replies"
+        item = data.get(key)
+        if isinstance(item, (string_types, dict)):
+            data[key] = (
+                []
+                if item == ""
+                else reddit._objector.objectify(item)._data["children"]
+            )
+
+        key = "subreddit"
+        item = data.get(key)
+        if isinstance(item, string_types):
+            data[key] = Subreddit(reddit, display_name=item)
+        elif isinstance(item, Subreddit):
+            item._reddit = reddit
```

Comment:
Similar question to what I asked before about why we need to set the `_reddit` instance in these cases.

Reply:
It’s only for solving issue #954. The PRAW codebase doesn’t otherwise use this code path.